articles/api/v2/changes.md

```diff
@@ -168,13 +168,13 @@ For example, to create a connection that is enabled for clients `AaiyAPdpYddboKn
 ```
 curl -H "Authorization: Bearer {API_TOKEN}" -X POST -H "Content-Type: application/json"
 -d '{"name":"new-connection","strategy":"auth0","enabled_clients":["AaiyAPdpYddboKnqNS8HJqRn4T5ti3BQ","DaM8bokEXBWrTUFZiXjWn50jei6ardyV"]}'
-https://login.auth0.com/api/v2/connections
+https://{YOU}.auth0.com/api/v2/connections
```

Comment:
Should be YOUR_TENANT rather than {YOU}. ie `https://YOUR_TENANT.auth0.com/api/v2/connections`